From 0c4e2af7711e97330fa2d2f51719c4d009b9fee6 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Mon, 30 Mar 2015 16:35:30 -0700
Subject: [PATCH 028/441] ACPI-LPSS: Add ACPI IDs for Cougar Mountain

Adding LPSS device IDs for Cougar Mountain. Specifically targeting I2C,
SPI, and UART.
---
 drivers/acpi/acpi_lpss.c | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/drivers/acpi/acpi_lpss.c
+++ b/drivers/acpi/acpi_lpss.c
@@ -199,6 +199,13 @@ static const struct acpi_device_id acpi_
 	{ "8086228E", (unsigned long)&byt_spi_dev_desc },
 	{ "80862288", }, /* CHT PWM */
 
+	/* Cougar Mountain LPSS devices */
+	{ "80862BC1", (unsigned long)&cht_i2c_dev_desc },
+	{ "80862B8A", (unsigned long)&byt_uart_dev_desc },
+	{ "80862B86", }, /* CGM DMA1 */
+	{ "80862BC0", }, /* CGM DMA2 */
+	{ "80862B8E", (unsigned long)&byt_spi_dev_desc },
+
 	{ }
 };
 
